CHANGES.md

```diff
@@ -1,20 +1,60 @@
 # Changelog
 
-## 1.10.2 (core) / 0.26.2 (libraries)
+## 1.10.3 (core) / 0.26.3 (libraries)
+
+### New
+
+- Added links from pool info in run event logs to the respective pool configuration pages.
+- Added queued run information on the pool info page, even if the pool granularity is set to `run`.
+- [ui] Added information about asset partitions that fail to materialize due to run cancellations to the asset partition detail page.
+- [ui] Added two new themes for users with reduced sensitivity to red and green light.
+- [ui] Added Not Diamond icon for asset `kinds` tag. (Thanks [@dragos-pop](https://github.com/dragos-pop)!)
+- [ui] Added Weaviate icon for asset `kinds` tag. (Thanks [@jjyeo](https://github.com/jjyeo)!)
+- [ui] Made Alerts page visible to users with Viewer roles.
+- [dagster-insights] Fix links to branch deployments in the deployment list UI.
+- [dagster-insights] Adjusted the way batching runs from the `create_snowflake_insights_asset_and_schedule` sensor using the `schedule_batch_size_hrs` parameter works to yield a single partition range run instead of individual runs per partition.
+
+### Bugfixes
+
+- Fixed a bug where a sensor emitting multiple `RunRequests` with the same run key within a single tick would cause two runs with the same key to be executed. Now, only the first run will be executed. (Thanks [@Gw1p](https://github.com/Gw1p)!)
+- Fixed a bug where run step selections were not affecting which pools limit a given run.
+- Fixed an issue where seeding the random number generator during code import or when initializing a resource could cause every step to write to the same stdout or stderr key.
+- [ui] Fixed an issue where certain jobs weren't showing the assets they targeted.
+- Asset backfills will now move into a `CANCELED` state instead of a `FAILURE` state when not every requested partition has been marked as materialized or failed by the backfill.
+- [dagster-dbt] Fixed a bug breaking `packaged_project_dir` since supporting `profiles_dir` in `DbtProject`.
+- Fixed an issue with `DbIOManagers` being unable to process subclasses of handled types.
+- [ui] Preserved asset selection filters when navigating folders in the asset catalog.
+- [ui] Corrected PostgreSQL SVG icon for asset `kinds` tag. (Thanks [@dragos-pop](https://github.com/dragos-pop)!)
+- [ui] Fixed an issue that caused Markdown with code blocks in languages not supported for syntax highlighting to crash the page.
+- Fixed an issue where asset backfills included failed partitions in the in-progress list in logging output.
+- Removed the cap on `PostgresEventLogStorage` `QueuePool` by setting [`max_overflow`](https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.create_engine.params.max_overflow) to `-1`. (Thanks [@axelwas](https://github.com/axelwas)!)
```

Comment:
This feels like more of a new capability than a bug fix?

Reply:
And I know we're not particularly consistent with the text tag, but maybe we clarify this is only for [dagster-postgres]